installation-tests/bin/npm_i

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+if [[ $1 == "i" || $1 == "install" ]]; then
+  shift;
+  args="install"
+  for i in "$@"; do
+    if [[ "$i" == "playwright" ]]; then
+      args="${args} ${PLAYWRIGHT_TGZ}"
+    elif [[ $i == "playwright-core" ]]; then
+      args="${args} ${PLAYWRIGHT_CORE_TGZ}"
+    elif [[ $i == "playwright-firefox" ]]; then
+      args="${args} ${PLAYWRIGHT_FIREFOX_TGZ}"
+    elif [[ $i == "playwright-chromium" ]]; then
+      args="${args} ${PLAYWRIGHT_CHROMIUM_TGZ}"
+    elif [[ $i == "playwright-webkit" ]]; then
+      args="${args} ${PLAYWRIGHT_WEBKIT_TGZ}"
+    elif [[ $i == "@playwright/test" ]]; then
+      args="${args} ${PLAYWRIGHT_TEST_TGZ}"
+    else
+      args="${args} $i"
+    fi
+  done
+  npm $args 2>&1
+else
+  npm $@ 2>&1
```

Comment:
Instead of running, let's error:

echo "Did you mean to use npm ${$@} instead?"